lib/IRGen/GenStruct.cpp

```diff
@@ -40,6 +40,7 @@
 #include "ResilientTypeInfo.h"
 #include "StructMetadataLayout.h"
 
+// TODO: address this missing override warning, don't ignore it.
```

Comment:
No, I tried once. It really doesn't make sense with the way these files are using overrides—they're coming from templates. (Or macros, I can't remember.)